standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2510,6 +2510,15 @@ struct GetAllWriteEventInfoRequest {
   3: optional string tableName
 }
 
+struct DeleteColumnStatisticsRequest {
+  1: required string cat_name,
```

Comment:
nit:  the `cat_name` should be optional to align with other request, the `engine` as well,
```
optional string engine = "hive"
```